vision/google/cloud/vision_helpers/decorators.py

```diff
@@ -99,20 +101,23 @@ def _create_single_feature_method(feature, enum):
     feature_value = {'type': enum.__dict__[feature]}
 
     # Define the function to be returned.
-    def inner(self, image, max_results=2**31-1,
+    def inner(self, image, max_results=None,
               retry=None, timeout=None, **kwargs):
         """Return a single feature annotation for the given image.
 
         Intended for use with functools.partial, to create the particular
         single-feature methods.
         """
-        feature_value['max_results'] = max_results
+        copied_features = copy.copy(feature_value)
```

Comment:
I see we need to copy because `feature_value` is a "shared global" (well, in the scope of the closure).  It would be simpler / faster / clearer just to use `dict.copy`, though, e.g.:

```python
    copied_features = feature_value.copy()
```

Reply:
done